pkg/api/update/update.go

```diff
@@ -0,0 +1,50 @@
+package update
+
+import (
+	"io"
+	"net/http"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	lock chan bool
+)
+
+// New is a factory function creating a new  Handler instance
+func New(updateFn func()) *Handler {
+	lock = make(chan bool, 1)
+	lock <- true
+
+	return &Handler{
+		fn:   updateFn,
+		Path: "/v1/update",
+	}
+}
+
+// Handler is an API handler used for triggering container update scans
+type Handler struct {
+	fn   func()
+	Path string
+}
+
+// Handle is the actual http.Handle function doing all the heavy lifting
+func (handle *Handler) Handle(w http.ResponseWriter, r *http.Request) {
+	log.Info("Updates triggered by HTTP API request.")
+
+	_, err := io.Copy(os.Stdout, r.Body)
```

Comment:
What's the purpose of this?

Reply:
Logging to stdout and verifying the request is not partial.

Reply:
@victorcmoura was there any specific reasoning behind this line of code or would you be ok with me deleting it? the info log just above is probably sufficient.

Reply:
@simskij, I agree that the log is sufficient, I believe you can delete it. Actually, I don't even remember writing this line of code haha, I didn't even remember I could io.Copy to stdout =P